docs/reference/commandline/service_create.md

```diff
@@ -875,6 +876,20 @@ $ docker inspect --format="{{.Config.Hostname}}" 2e7a8a9c4da2-wo41w8hg8qanxwjwsg
 x3ti0erg11rjpg64m75kej2mz-hosttempl
 ```
 
+### Specify isolation (Windows)
+
+By default, tasks scheduled on Windows nodes are run using the default isolation mode configured for this particular node. If you want to force a specific isolation, you can use the `--isolation` flag: 
+
+```bash
+$ docker service create --name processisolated --isolation=process microsoft/nanoserver
```

Comment:
nit: `processisolated` looks funny, perhaps just `myservice` (also makes it clearer there's nothing "special" in this value)